python/desc/sims/GCRCatSimInterface/TwinklesClasses.py

```diff
@@ -0,0 +1,202 @@
+import os
+import numpy as np
+from desc.twinkles.twinklesVariabilityMixins import VariabilityTwinkles
+from desc.twinkles import sprinklerCompound
+from lsst.sims.catUtils.exampleCatalogDefinitions.phoSimCatalogExamples import PhoSimSpecMap as psmp
+from lsst.sims.catalogs.db import CompoundCatalogDBObject
+from desc.twinkles import TwinklesCompoundInstanceCatalog, sprinkler
+from . import CompoundDESCQAInstanceCatalog, GalaxyCompoundDESCQAObject, PhoSimDESCQA, PhoSimDESCQA_AGN
+
+
+twinkles_sn_sed_dir = 'spectra_files'
+twinkles_spec_map = psmp
+twinkles_spec_map.subdir_map['(^specFile_)'] = twinkles_sn_sed_dir
+
+_twinkles_defs_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_defs.csv')
+
+_agn_cache_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_agn_cache.csv')
+_sne_cache_file = os.path.join(os.environ['TWINKLES_DIR'], 'data', 'dc2_sne_cache.csv')
+global_agn_params_db = os.path.join('/global/cscratch1/sd/brycek/',
+                                    'proto_dc2_agn', 'test_agn.db')
+
+assert os.path.exists(global_agn_params_db)
+assert os.path.exists(_agn_cache_file)
+assert os.path.exists(_sne_cache_file)
+
+__all__ = ["TwinklesCompoundInstanceCatalog_DC2",
+           "sprinklerCompound_DC2", "TwinklesCatalogSersic2D_DC2",
+           "TwinklesCatalogZPoint_DC2"]
+
+
+#class sprinklerCompound_DC2(sprinklerCompound, GalaxyCompoundDESCQAObject):
+class sprinklerCompound_DC2(GalaxyCompoundDESCQAObject):#, sprinklerCompound):
+    objid = 'sprinklerCompound_DC2'
+    objectTypeId = 166
+    cached_sprinkling = True
+    agn_cache_file = _agn_cache_file
+    sne_cache_file = _sne_cache_file
+    defs_file = _twinkles_defs_file
+
+    agn_params_db = global_agn_params_db
+    agn_objid = 'agn_descqa'
+
+    def _final_pass(self, results):
+        #From the original GalaxyTileCompoundObj final pass method
+        for name in results.dtype.fields:
+            if 'raJ2000' in name or 'decJ2000' in name:
+                results[name] = np.radians(results[name])
+
+        # the stored procedure on fatboy that queries the galaxies
```

Comment:
Go ahead and remove this block of comments, since this repo should never be used to get galaxies from fatboy